src/Raven.Studio/wwwroot/App/views/resources/serverDashboard.html

```diff
@@ -1,4 +1,4 @@
-<div id="dashboard-container" class="content-margin">
+<div id="dashboard-container" class="content-margin cluster">
```

Comment:
it isn't cluster page, Why cluster class here?

Reply:
removed